doc/man1/openssl-sess_id.pod

```diff
@@ -18,10 +18,10 @@ B<openssl> B<sess_id>
 
 =head1 DESCRIPTION
 
-The B<sess_id> process the encoded version of the SSL session structure
-and optionally prints out SSL session details (for example the SSL session
-master key) in human readable format. Since this is a diagnostic tool that
-needs some knowledge of the SSL protocol to use properly, most users will
+This command command processes the encoded version of the SSL session
```

Comment:
command command

Reply:
It's the super-command!

Nah, ok, will fix